src/framework/theme/service/styleUtil.service.ts

```diff
@@ -56,52 +59,42 @@ const FALLBACK_MAPPING_VARIANT: MappingType = {};
  * @param states: string[] - states in which component is. Default is []
  *
  * @return StyleType - compiled component styles declared in mappings, mapped to theme values
- *
  */
 export function createStyle(theme: ThemeType,
                             mapping: ComponentMappingType,
                             appearance: string = APPEARANCE_DEFAULT,
                             variants: string[] = [],
                             states: string[] = []): StyleType {
 
-  const appearanceMapping = createAppearanceMapping(
-    mapping,
-    normalizeAppearance(appearance),
-    normalizeVariants(variants),
-    normalizeStates(states),
-  );
-  return createStyleFromMapping(appearanceMapping, theme);
-}
+  const appearanceDescription = normalizeAppearance(appearance);
```

Comment:
why not `normalizedAppearance`?

Reply:
`normalizedAppearance` is a sort of variable name, while `normalizeAppearance` is an action meaning that we're going to normalize something. Which means it is a function